### ISSUE TITLE
Go to Step Declaration from .feature file

### ISSUE THREAD

**Issue by vrnvorona** [jonsamwell/flutter_gherkin#88]
Hello, i have a question: is it possible to go to step declaration from feature file via shortcut? Ruby cucumber supports this behavior: by cmd/ctrl-clicking on step in Gherkin code, you get into your feature file where definition of that step is declared.

Perhaps a plugin like Ogurets https://github.com/dart-ogurets/OguretsIntellij?

**Comment by jonsamwell** [comment]
Yes this would be really helpful.  I have been meaning to do a vscode
plugin for this for a while now, it is just finding the time to do it. So
help would be appreciated!

**Comment by vrnvorona** [comment]
Well i am working with IntelliJ. I have barely any experience with Java, but i will look what i can do in free time.

**Comment by jonsamwell** [comment]
Great thanks, I have never used IntelliJ but I can do a plugin for VSCode if I get some time.

**Comment by gwanhyeong** [comment]
@jonsamwell 

Is there any way to use this extension?

https://marketplace.visualstudio.com/items?itemName=alexkrechik.cucumberautocomplete

**Comment by gwanhyeong** [comment]
Hello, @jonsamwell 

This extension is compatible.
[[VSCode Extension] Cucumber (Gherkin) Full Support](https://marketplace.visualstudio.com/items?itemName=alexkrechik.cucumberautocomplete)

It doesn't support absolute routes, so library hooks are impossible.

`.vscode/settings.json`
```json
{
  "cucumberautocomplete.steps": ["integration_test/gherkin/steps/*.dart"],
  "cucumberautocomplete.smartSnippets": true,
  "cucumberautocomplete.strictGherkinCompletion": true,
  "cucumberautocomplete.strictGherkinValidation": true,
  "cucumberautocomplete.gherkinDefinitionPart": "(given|when|then)(?:\\d?.*>|\\d)\\(",
  "cucumberautocomplete.customParameters": [
    {
      "parameter": "RegExp(r'",
      "value": "('"
    },
  ],
  "cucumberautocomplete.stepRegExSymbol": "'"
}
```

And I haven't found a way to solve the regular expression.
```Dart
return then1<String, FlutterWorld>(
    RegExp(r'I tap the (?:button|element|label|field|text|widget) ...'),
```
